File: segundo-semestre/lab121/c_sharp/dia_caso.cs
class MainClass {
  public static void Main(string[] args) {
    System.Console.WriteLine("Introduce un numero del i al 7: ");
    int number = System.Convert.ToInt32(System.Console.ReadLine());

    if (number >= 1 && number <= 7) {
      switch (number) {
      case 1:
        System.Console.WriteLine("Lunes");
        break;
      case 2:
        System.Console.WriteLine("Martes");
        break;
      case 3:
        System.Console.WriteLine("Miercoles");
        break;
      case 4:
        System.Console.WriteLine("Jueves");
        break;
      case 5:
        System.Console.WriteLine("Viernes");
        break;
      case 6:
        System.Console.WriteLine("Sabado");
        break;
      case 7:
        System.Console.WriteLine("Domingo");
        break;
      }
    }
  }
}

File: segundo-semestre/lab121/tarea1/java/dia_caso.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner reader = new Scanner(System.in);
    System.out.println("Introduce un numero del 1 al 7: ");
    int number = reader.nextInt();

    if (number >= 1 && number <= 7) {

      switch (number) {
      case 1:
        System.out.println("Lunes");
        break;
      case 2:
        System.out.println("Martes");
        break;
      case 3:
        System.out.println("Miercoles");
        break;
      case 4:
        System.out.println("Jueves");
        break;
      case 5:
        System.out.println("Viernes");
        break;
      case 6:
        System.out.println("Sabado");
        break;
      case 7:
        System.out.println("Domingo");
        break;
      }
    }
  }
}
